client/post-editor/editor-drawer/featured-image.jsx

```diff
@@ -41,11 +42,13 @@ class EditorDrawerFeaturedImage extends Component {
 					empty={ ! site || ! post || ! getFeaturedImageId( post ) }
 					onClick={ this.startSelecting }
 					onRemove={ this.removeImage }>
-					<FeaturedImage
-						selecting={ this.state.isSelecting }
-						onImageSelected={ this.endSelecting }
-						site={ site }
-						post={ post } />
+					<MediaListData siteId={ this.props.site.ID }>
```

Comment:
Yep, It is, I've already used it in Stats. I think it should be fine to use. The only issue we could have is if we're expecting the Flux Store to be filled with the media somewhere on the underlying component.